packages/appstore/src/components/main-title-bar/account-type-dropdown.tsx

```diff
@@ -15,7 +16,7 @@ const AccountTypeDropdown = () => {
         <div className={classNames('account-type-dropdown--parent')}>
             <Dropdown
                 classNameIcon={`account-type-dropdown__icon--${selected_account_type}`}
-                value={selected_account_type}
+                value={localize({ selected_account_type })}
```

Comment:
It is not possible to directly pass variables in the localize function. What you might have to do is use the `Localize` component and pass the values through there, using the values parameter. Only with the `Localize` component function can we pass variables, `localize()` allows only strings.

I think this might work:

```
...
value={
  <Localize
      i18n_default_text='{{ selected_account_type }}'
      values={{ selected_amount_type }}
  />
...
}
```

I suggest to have a look on how others have implemented Localize components in this project/repository.